src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -36,6 +37,36 @@ export type SurvivalPlotFilters = {
     y: [number, number];
 };
 
+// TODO should remove this if we want to be generic
+export const survivalClinicalDataVocabulary: { [prefix: string]: string[] } = {
+    OS: ['DECEASED'],
+    PFS: ['CENSORED'],
+    DFS: ['Recurred/Progressed', 'Recurred'],
+    DSS: ['DEAD WITH TUMOR'],
+    EFS: ['Censored'],
+};
+
+// TODO should remove this if we want to be generic
+export const survivalPlotTooltipxLabelWithEvent: {
+    [prefix: string]: string;
+} = {
+    // TODO text for PFS DSS EFS?
+    OS: 'Time of death',
+    PFS: 'Time of death',
+    DFS: 'Time of Relapse',
+    DSS: 'Time of death',
+    EFS: 'Time of death',
+};
```

Comment:
@jjgao what text would be good for PFS, DSS and EFS? this text will show in the tooltip as an explanation of x label.
![image](https://user-images.githubusercontent.com/16869603/76232167-b2583700-61fc-11ea-92fe-683c341eaef9.png)